### ISSUE TITLE
[Bug]: Accessibility Issue with  Details List Component

### ISSUE THREAD

**Issue by bhawnakumarigupta** [microsoft/fluentui#27232]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Environment Information
Package version(s): Fluent UI React 8.106.2
Browser and OS versions: Microsoft Edge Browser Windows 11


```

### Are you reporting Accessibility issue?

Yes

### Bug Description
**Describe the issue:**

On details list component if we run the Accessibility insights tool, we are getting error.
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

## Actual Behavior
![image](https://user-images.githubusercontent.com/128052223/225623102-ba741534-85d9-425c-a092-2f2019777342.png)

![image](https://user-images.githubusercontent.com/128052223/225623617-0ed3c575-fc3b-4f67-944c-0ded53c159e0.png)

## Expected Behavior
Bug should not be reported for this.

**Comment by behowell** [comment]
This might be a false positive by Accessibility Insights. @smhigley can you confirm whether this is a legit bug or a false positive?

The rule that's failing is [aria-required-children](https://accessibilityinsights.io/info-examples/web/aria-required-children/), which says:

> This rule passes if ALL of the following are true:
> * At least one child has an ARIA role which is one of the parent ARIA role's [Required Owned Elements](https://www.w3.org/TR/wai-aria/#mustContain).
> * All children either:
>     * have an ARIA role which is one of the parent ARIA role's [Required Owned Elements](https://www.w3.org/TR/wai-aria/#mustContain), OR
>     * Are not present in the browser's accessibility tree

The elements that it's complaining about are the DetailsList's cellSizerStart elements, which have `role="button"`, and so aren't valid children of the `role="row"` container. However, they _also_ have `aria-hidden="true"`, so they shouldn't be in the accessibility tree, and as far as I can tell shouldn't be causing this failure.


Here's the full failure log in case it helps:

> Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (div[data-focuszone-id="FocusZone179"])
> Tags: Accessibility, WCAG 1.3.1, aria-required-children
> 
> Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)
> 
> Target application: Fluent UI - Controls - React - DetailsList - https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist
> 
> Element path: div[data-focuszone-id="FocusZone179"]
> 
> Snippet: <div role="row" class="ms-FocusZone css-134 ms-DetailsHeader root-297" data-automationid="DetailsHeader" data-focuszone-id="FocusZone179">
> 
> Related paths: 
>   div[data-focuszone-id="FocusZone179"] > .cellSizerStart-305.ms-DetailsHeader-cellSizer[data-sizer-index="1"]
>   div[data-focuszone-id="FocusZone179"] > .cellSizerStart-305.ms-DetailsHeader-cellSizer[data-sizer-index="2"]
>   .cellSizerStart-305.ms-DetailsHeader-cellSizer[data-sizer-index="3"]
>   div[data-sizer-index="4"]
> 
> How to fix: 
> Fix any of the following:
>   Element has children which are not allowed (see related nodes)
>   Element has no aria-busy="true" attribute
> 
> Environment: Microsoft Edge version 111.0.1661.54
> 
> ====
> 
> This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by mrtalley** [comment]
@behowell @smhigley Any update on this? I'm running into this with the `TabList` component mentioned in [#27275](https://github.com/microsoft/fluentui/issues/27275).

**Comment by behowell** [comment]
Resolving as External, since this is a bug in Accessibility Insights and not FluentUI (see @smhigley's comment above).
